viime/models.py

```diff
@@ -66,6 +67,25 @@ def make_object(self, data, **kwargs):
         return data
 
 
+class GroupLevel(db.Model):
```

Comment:
A GroupLevel sounds like a quantitative piece of information, like a threshold.  What does GroupLevel mean?

Reply:
coming from the R world: if you want to get the different unique values of a factor you call `levels()`

see also https://www.rdocumentation.org/packages/base/versions/3.6.1/topics/levels and https://www.rdocumentation.org/packages/base/versions/3.6.1/topics/factor

Reply:
Ah neat thanks!